extensions/wikia/Blogs/BlogsHelper.class.php

```diff
@@ -120,4 +120,31 @@ public static function onFilePageImageUsageSingleLink( &$link, &$element ) {
 		wfProfileOut( __METHOD__ );
 		return true;
 	}
+
+	/**
+	 * Enables commenting for a blog post that has been moved from another namespace
+	 * @param object Title $oOldTitle An object for the old article's name
+	 * @param object Title $oNewTitle An object for the new article's name
+	 * @param object User $oUser
```

Comment:
You don't need the `object` on these three, the types should be just `Title` and `User`.